### DISCUSSION TITLE
nuxt ssg ,How to not put usefetch data in nuxt_ In data

### DISCUSSION THREAD

**Discussion by zbk-it** [nuxt/nuxt#23168] [General]
How to not write the data retrieved by the server to HTML (<script type="application/JSON" id="__NUXT-DATA__" data ssr="true">xxxx</script>) during SSG? This data can be removed?

Is this data the entire website, the current page, or the current spa

**Comment by logotip4ik** [comment]
You can disable including data in payload by disabling `useFetch` call on the server (`server: false`), but that probably not what you wanted. Also you probably could manually remove not needed contents, but that will cause hydration issues :(

**Reply by gllinzbk1** [reply]
Nuxt data, Is the static page spa generated by SSG requesting server data for the first time a duplicate request for data

My current situation is SSG, which generates HTML and also requests server-side data
@logotip4ik @victortolbert

**Reply by septatrix** [reply]
> You can disable including data in payload by disabling `useFetch` call on the server (`server: false`), but that probably not what you wanted. Also you probably could manually remove not needed contents, but that will cause hydration issues :(

Is the opposite also possible? I too use Nuxt for SSG but do not need to refresh the data on the client (see #26873)
